### PR TITLE
fixes missing semicolon in comment.

### PR DESCRIPTION
**Pull request by bcoe**

### CODE REVIEW

**Comment by alexweber** [comment]
:+1:

**Comment by pygy** [comment]
That breaks the right hand side alignment, though...

File: semicolons.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-;// require('semicolons') ==> they are REQUIRED!;
+;// require('semicolons'); ==> they are REQUIRED!;
```
Comment (brock) [inline semicolons.js]: change `==>` to `=>` and this will be good.

**Comment by isaacs** [comment]
That's not a missing semicolon.  They're only required at the end of lines, not in the middle.

**Comment by isaacs** [comment]
I'm not even sure that's safe.

**Pull request closed without merging**